smartsim/entity/dbobject.py

```diff
@@ -196,6 +196,8 @@ def __init__(
         if not script and not script_path:
             raise ValueError("Either script or script_path must be provided")
 
+        self.script_path = script_path
```

Comment:
Curious why we need this defined here when it looks like it is on the parent object: ``DBObject``

Reply:
Removed this